dev/com.ibm.ws.st.common.core.ext/src/com/ibm/ws/st/common/core/ext/internal/util/BaseDockerContainer.java

```diff
@@ -373,8 +374,11 @@ public static String getHostFromMapping(String mapping) {
         Map<String, String> bindings = new HashMap<String, String>();
         JsonObject hostConfig = (JsonObject) inspect.get(HOST_CONFIG_KEY);
         if (hostConfig != null) {
-            JsonObject portBindings = (JsonObject) hostConfig.get(PORT_BINDINGS_KEY);
-            if (portBindings != null) {
+            JsonValue jsonValue = hostConfig.get(PORT_BINDINGS_KEY);
+            // If jsonValue is an instance of ValueType, then simply return an empty map because there are no port bindings defined.
```

Comment:
Can the comment be changed to "If jsonValue is not an instance of JsonObject, ..."?